### ISSUE TITLE
[https://donate.mozilla.org] Footer tweaks

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/donate-wagtail#1017]
### Background:
We will add a new link to the footer that will take users to the "Press Center" page - here is a [mockup of the page](https://ia-refresh.mofostaging.net/en/press-center/) in our IA environment. With this change there is an opportunity to make all our footers consistent.

### Tasks:
• Add the "Press Center" footer label to https://donate.mozilla.org
• Add the label "Cookies" that is missing from this page
• Remove "Donate" label from the footer since people are already on this page.
• Reorder the items by alphabetical order (that works on Desktop, Tablet and Mobile views).
Note: I checked with Will first to approve these changes and we got the thumbs up for those. 

### Dependencies:
We need to create the Press Center page in prod and for this to happen we need the page [described here](https://github.com/mozilla/foundation.mozilla.org/issues/4777)

### Desired final state:
<img width="1400" alt="image" src="https://user-images.githubusercontent.com/22157921/84539361-18df6c00-aca8-11ea-80f3-920ab567ad9c.png">

@TheoChevalier tagging you here as this may have implications for localized pages. Kevin mentioned in another ticket that this page should be kept in English so people know that who will read it is an English speaker. However we may consider localize the label "Press Center" in all languages we have the footer on.

**Comment by taisdesouzalessa** [comment]
**Note:** let's wait on [ticket 4868 (from foundation.mozilla.org repo)](https://github.com/mozilla/foundation.mozilla.org/issues/4868) to be resolved then open a new ticket with all footer changes at once.
